build.sbt

```diff
@@ -0,0 +1,29 @@
+import Dependencies._
+
+ThisBuild / scalaVersion := "2.13.1"
+ThisBuild / version := "0.1.0-SNAPSHOT"
+ThisBuild / organization := "com.codacy"
+ThisBuild / organizationName := "codacy"
+
+val graalVersion = "19.3.1-java11"
+
+lazy val root = (project in file("."))
+  .enablePlugins(GraalVMNativeImagePlugin)
+  .settings(
+    name := "codacy-clang-tidy",
+    libraryDependencies ++= Seq(pluginsApi) ++ circe,
+    libraryDependencies += scalaTest % Test,
+    scalacOptions += "-Ywarn-macros:after", // checks for unused implicits after macro expansion
+    // Graal vm build options
+    graalVMNativeImageGraalVersion := Some(graalVersion),
+    containerBuildImage := Some(s"oracle/graalvm-ce:$graalVersion"),
+    graalVMNativeImageOptions ++= Seq(
+      "-O1",
+      "-H:+ReportExceptionStackTraces",
+      "--no-fallback",
+      "--no-server",
+      "--initialize-at-build-time",
+      "--report-unsupported-elements-at-runtime",
+      "--static"
```

Comment:
This is nice 👍

Reply:
I know!! It's awesome and so simple

Reply:
Is this the sbt-native-packager? Does it depend on a docker?

Reply:
Yes and yes.